### DISCUSSION TITLE
Numeric _value

### DISCUSSION THREAD

**Discussion by zembrowski** [spatie/array-to-xml#213] [Q&A] [answered]
What is the reason a `_value` has to be a string in [ArrayToXml.php#L157](https://github.com/spatie/array-to-xml/blob/cf5f4e48019be97e9f1f6921666bd8957a4c4c86/src/ArrayToXml.php#L157)?

Passing an integer:
```JSON
"_value": 1
```

Produces a new node `<_value>1</_value>`.

The linked [issue #75](https://github.com/spatie/array-to-xml/issues/75#issuecomment-413726065) from the Readme seems not be accessible anymore. I found the linked page in the [Wayback Machine](https://web.archive.org/web/20200920131332/https://github.com/spatie/array-to-xml/issues/75), but it lacks further explanation.

**Comment by gigerIT** [comment]
We have the same issue.
We're setting _value with strval(123456) or (string)123456 as workaround for now.

**Comment by dimitri-koenig** [comment, marked as answer]
Should be resolved with this PR: https://github.com/spatie/array-to-xml/pull/248
